libs/ui-lib/lib/ocm/components/clusterConfiguration/ExternalPlatformDropdown.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import { Dropdown, DropdownItem, DropdownToggle, FormGroup } from '@patternfly/react-core';
+import { CaretDownIcon } from '@patternfly/react-icons';
+import { useField } from 'formik';
+import {
+  NUTANIX_CONFIG_LINK,
+  PlatformType,
+  VSPHERE_CONFIG_LINK,
+  getFieldId,
+} from '../../../common';
+
+const INPUT_NAME = 'platform';
+const fieldId = getFieldId(INPUT_NAME, 'input');
+export type ExternalPlatformType = Extract<PlatformType, 'none' | 'nutanix' | 'oci' | 'vsphere'>;
+
+type ExternalPlatformDropdownProps = {
+  showOciOption: boolean;
+  selectedPlatform: ExternalPlatformType;
+  disabledOciTooltipContent: React.ReactNode;
+  isOciDisabled: boolean;
+  onChange: (isOracleOptionSelected: boolean) => void;
+};
+
+export type ExternalPlatformInfo = {
+  label: string;
+  href: string;
+  tooltip: string;
+};
+
+export const externalPlatformTypes: Record<ExternalPlatformType, ExternalPlatformInfo> = {
+  none: {
+    label: 'None',
+    href: '',
+    tooltip: '',
+  },
+  nutanix: {
+    label: 'Nutanix',
+    href: NUTANIX_CONFIG_LINK,
+    tooltip: '',
+  },
+  oci: {
+    label: 'Oracle  (Using custom manifests)',
+    href: 'www.google.es',
+    tooltip:
+      "To integrate with an external partner (for example, Oracle Cloud), you'll need to provide a custom manifest.",
+  },
+  vsphere: {
+    label: 'vSphere',
+    href: VSPHERE_CONFIG_LINK,
+    tooltip: '',
+  },
+};
+
+export const ExternalPlatformDropdown = ({
+  showOciOption,
+  selectedPlatform,
+  disabledOciTooltipContent,
+  isOciDisabled,
+  onChange,
+}: ExternalPlatformDropdownProps) => {
+  const [field, { value }, { setValue }] = useField<string>(INPUT_NAME);
+  const [currentPlatform, setCurrentPlatform] = React.useState(
+    externalPlatformTypes[selectedPlatform].label,
+  );
+  const [isOpen, setOpen] = React.useState(false);
+  const enabledItems = Object.keys(externalPlatformTypes)
+    .filter((platformType) => {
+      if (platformType === 'oci') {
+        return showOciOption;
+      }
+      return true;
+    })
+    .map((platformType) => {
+      const { label, href, tooltip } = externalPlatformTypes[platformType as ExternalPlatformType];
+      const isOracleDisabled = platformType === 'oci' && isOciDisabled;
```

Comment:
```suggestion
      const isDisabledPlatform = platformType === 'oci' && isOciDisabled;
```
since the boolean is applied to all items

Reply:
no, it's only for oci option this boolean